js/views/whisper_view.js

```diff
@@ -54,7 +54,7 @@
                     resolve: resolve,
                     reject: reject
                 });
-                this.$el.closest('body').append(dialog.el);
```

Comment:
That's the initial fix. I explained it above. If the dialog appends directly under the body element, the styles aren't effective.